### ISSUE TITLE
Makefile: set SHELL explicitly

### ISSUE THREAD

**Issue by jelmd** [spechub/Hets#1524]
Makefile should set the SHELL explicitly to avoid suprises. E.g. since Ubuntu has changed the default shell from ~~bash~~ to **dash** on recent distros, things like 

```
$(RM) test/*/*.{thy,pp.het,pp.tex,th,dfg.c,xml,log,dvi,aux,sty}
```

do not work anymore, since dash seems to not support curly braces expansions (unlike ksh*, bash, *csh, ...) as well as a lot of other useful stuff.

So IMHO it is the best to put a `SHELL = /bin/bash` (or ksh93 ;-) ) on top of the main Makefile (and probably on all others, which can be used standalone and make use of such features) to avoid further suprises. Just setting the SHELL or MAKESHELL env var to the needed value - as the make documentation suggestes - doesn't seem to work (tried several combinations incl. -e etc.).

**Comment by cmaeder** [comment]
can `SHELL = /bin/bash` be set in `var.mk`?

**Comment by cmaeder** [comment]
Are you on `wily` already?  (They'll break a lot of scripts)

**Comment by cmaeder** [comment]
The alternative is to change the code (into a for loop).

**Comment by jelmd** [comment]
Yepp, var.mk is sufficient AFAICS.

Wily? no. I'll try, when the required artificats are available...

loop/find alternative: makes it IMHO less readable (just more bloat)...

**Comment by jelmd** [comment]
fixed via #1830
